html/php-pre-challenge2/index.php

```diff
@@ -2,9 +2,21 @@
 $array = explode(',', $_GET['array']);
 
 // 修正はここから
-for ($i = 0; $i < count($array); $i++) {
-
+$count = count($array);
+for ($i = 0; $i < $count; $i++) {
+    for($j = $i+1; $j <$count; $j++){
+        //$iの値が$jの値より大きければ
+        if ($array[$i] > $array[$j]){
+            //$tmpに$iを一旦代入
+            $tmp = $array[$i];
+            //$iに$jを代入して入れ替える
+            $array[$i] = $array[$j];
+            //$jに$tmpを代入
+            $array[$j] = $tmp; 
```

Comment:
```suggestion
            $array[$j] = $tmp;
```
末尾の半角スペースを除去しましょう。

vscodeのプラグイン php intelephense を入れるとスタイルを丸ごと自動整形してくれるのでお勧めです

![image](https://user-images.githubusercontent.com/22877094/84624173-d8882400-af1b-11ea-9b5d-c2eb3bfc56bf.png)

Reply:
おすすめしていただき以前インストールしたのですが設定で自動整形がonになっていなかったため効いていないようでした。先ほど設定を変更しonにしました。ありがとうございます！